android/KMEA/app/src/main/java/com/tavultesoft/kmea/cloud/CloudDataJsonUtil.java

```diff
@@ -153,9 +153,10 @@ public static void processKeyboardPackageUpdateJSON(Context aContext, JSONObject
             for (int i = 0; i < KeyboardController.getInstance().get().size(); i++) {
               Keyboard kbd = KeyboardController.getInstance().getKeyboardInfo(i);
               String version = kbd.getVersion();
+              String updateKMP = kbd.getUpdateKMP();
               if (keyboardID.equalsIgnoreCase(kbd.getKeyboardID()) &&
                   (FileUtils.compareVersions(cloudVersion, version) == FileUtils.VERSION_GREATER) &&
```

Comment:
```suggestion
                  FileUtils.compareVersions(cloudVersion, version) == FileUtils.VERSION_GREATER &&
```

Reply:
I'll isolate this change to a separate PR #7787